deploy/iso/minikube-iso/package/crio-bin/crio.conf.default

```diff
@@ -202,15 +217,19 @@ gid_mappings = ""
 # regarding the proper termination of the container.
 ctr_stop_timeout = 0
 
-  # The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.
-  # The runtime to use is picked based on the runtime_handler provided by the CRI.
-  # If no runtime_handler is provided, the runtime will be picked based on the level
-  # of trust of the workload.
-  
-  [crio.runtime.runtimes.runc]
-  runtime_path = "/usr/bin/runc"
-  runtime_type = "oci"
-  
+# ManageNetworkNSLifecycle determines whether we pin and remove network namespace
+# and manage its lifecycle.
+manage_network_ns_lifecycle = false
+
+# The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.
+# The runtime to use is picked based on the runtime_handler provided by the CRI.
+# If no runtime_handler is provided, the runtime will be picked based on the level
+# of trust of the workload.
+
+[crio.runtime.runtimes.runc]
+runtime_path = ""
```

Comment:
Please be aware that `runc` has to be in `$PATH` to let this work during runtime.

Reply:
Yeah, this is what the crio generated by default. Compare it with our config (above).

    runtime_path = "/usr/bin/runc"

I generated and added the default config, since it changes and breaks _all. the. time_

* plugin_dir, plugin_dirs

Reply:
I guess it should work fine :)